index.html

```diff
@@ -71,9 +71,13 @@
 are starting to be developed and deployed.
 A large number of jurisdictions are starting
 to create digital identity systems for their citizens.
+
 The [Federated Identity Working Group](https://www.w3.org/2025/02/wg-fedid.html)
 includes an API for accessing these systems
 as a chartered deliverable.
+This API follows a three-party model
+where the primary actors are issuer, holder, and verifier;
```

Comment:
#links and grammar nit:

```suggestion
where the primary actors are the [=issuer=], [=holder=], and [=verifier=];
```